packages/cascara/src/private/ViewConfig/ViewConfigBase.js

```diff
@@ -56,6 +56,8 @@ const ViewConfig = ({
   const renderOptions = (options, isSelected) =>
     options.map((option) => {
       const { currentSelection, ...setterFunctions } = state;
+      // isEditable should not propapate to MemoViewConfigItem component
+      delete option.isEditable;
       return (
         <MemoViewConfigItem
           {...option}
```

Comment:
```suggestion
          option={option}
```
Essentially, the original option would become a prop instead of getting spread into the ViewConfigItem. Then we can handle those values differently inside ViewConfigItem.